### PR TITLE
[5.1] Escape triple bracket echos

### PR DESCRIPTION
**Pull request by eklundchristopher**
I came across a scenario using vue.js where I needed to return unescaped HTML from a data property (http://vuejs.org/guide/index.html#Mustache_Bindings), and to do so, I need to use the triple bracket syntax (`{{{ var }}}`), but since Blade also uses that syntax, it was an obvious clash.

Seeing as how the double bracket syntax (`{{ $var }}`) allows you to escape it by prefixing it with an `@` symbol, I thought we might as well allow it for the triple syntax for whenever the need arises. (Unless there is a specific reason for it not being allowed already?)

So, in short, this PR allows for the following syntaxes:
`{{{ $var }}}` outputs `<?php echo e($var); ?>`
`@{{{ $var }}}` outputs `{{{ $var }}}`

### CODE REVIEW

**Comment by plashenkov** [comment]
It looks like this PR does not cut the "@" sign which is used to escape brackets. I.e. @ will go to the output.
